addon/components/o-s-s/attributes-panel.hbs

```diff
@@ -13,7 +13,7 @@
         {{#if (has-block "contextual-action")}}
           {{yield to="contextual-action"}}
         {{/if}}
-        <OSS::Button @icon="fa-pen" @square={{true}} {{on "click" this.toggleMode}}
+        <OSS::Button @icon={{this.editIcon}} @square={{true}} {{on "click" this.onEdit}}
```

Comment:
can you also precise the `size` ? we did something a while ago that makes square buttons small by default but that's not really explicit